qutebrowser/config/websettings.py

```diff
@@ -238,6 +238,25 @@ def _set(self, value, qws=None):
         self._setter(*args)
 
 
+class ThirdPartyCookies(Base):
+
+    """The ThirdPartyCookiePolicy setting is different from other settings."""
+
+    mapping = (
+        ('always', QWebSettings.AlwaysAllowThirdPartyCookies),
+        ('never', QWebSettings.AlwaysBlockThirdPartyCookies),
+        ('existing', QWebSettings.AllowThirdPartyWithExistingCookies),
+    )
+
+    def get(self, qws=None):
+        policy = QWebSettings.globalSettings().thirdPartyCookiePolicy()
+        return tuple(filter(lambda i: i[1] == policy, self.mapping))[0][0]
```

Comment:
(See the comment below)

Here I suggest doing something like `reverse_mapping = {v: k for k, v in mapping.items()}` (a [dict comprehension](https://www.python.org/dev/peps/pep-0274/)) at class level and then using `self.reverse_mapping[policy]`.